packages/connector-ethereum/src/index.ts

```diff
@@ -8,7 +8,7 @@ class ConnectorEthereum implements ConnectorInterface {
 
   constructor({}: ConnectorEthereumConfig) {}
 
-  async permissions(orgAddress: string): Promise<Permission[]> {
+  async permissionsForOrg(orgAddress: string): Promise<Permission[]> {
```

Comment:
I wonder if on the high order data (apps, permissions) we still want to add `ForOrg` after the name

Reply:
Yeah, this is the "controversial" bit that I mentioned in our conversations. It's something we seem to need on ConnectorInterface, but it doesn't feel too good.